psql2mysql/__init__.py

```diff
@@ -442,6 +450,9 @@ def main():
                         check_target_schema(db['target'])
                     cfg.CONF.command.func(cfg, db['source'], db['target'])
 
+        except Psql2MysqlRuntimeError as e:
+            print(e, file=sys.stderr)
```

Comment:
Mainly because I want to avoid spitting out python backtraces on the console for these specific (obvious) errors. Because these tent to confuse "normal" users. But maybe I am misunderstanding what you're suggesting. Could you please clarify?

Reply:
oh I see, so no big backtraces here then only the error message. Ok sounds good to me (although personally I like to see where the error comes from as to quickly pinpoint where in the code it failed)